openlibrary/coverstore/archive.py

```diff
@@ -317,30 +317,25 @@ def timestamp(self):
         )
         return time.mktime(t.timetuple())
 
-    def has_valid_files(self):
-        return all(f.path and os.path.exists(f.path) for f in self.files.values())
-
-    def get_files(self):
-        files = {
-            'filename': web.storage(name="%010d.jpg" % self.id, filename=self.filename),
-            'filename_s': web.storage(
-                name="%010d-S.jpg" % self.id, filename=self.filename_s
-            ),
-            'filename_m': web.storage(
-                name="%010d-M.jpg" % self.id, filename=self.filename_m
-            ),
-            'filename_l': web.storage(
-                name="%010d-L.jpg" % self.id, filename=self.filename_l
-            ),
-        }
-        for file_type, f in files.items():
-            files[file_type].path = f.filename and os.path.join(
+    @cached_property
```

Comment:
I'm not sure if we want this cached -- if we use in multiple locations within the code, we're probably calling it because we want the values to be different (e.g. if a file is renamed or moved we want it reflected in `files`)